youtube_dl/extractor/kaltura.py

```diff
@@ -103,6 +103,11 @@ class KalturaIE(InfoExtractor):
         {
             'url': 'https://www.kaltura.com:443/index.php/extwidget/preview/partner_id/1770401/uiconf_id/37307382/entry_id/0_58u8kme7/embed/iframe?&flashvars[streamerType]=auto',
             'only_matching': True,
+        },
+        {
+            # some GDC Vault pages embed a URL with extra leading space
+            'url': ' https://cdnapisec.kaltura.com/p/1670711/sp/167071100/embedIframeJs/uiconf_id/43558772/partner_id/1670711?iframeembed=true&playerId=kaltura_player_1547062087&cache_st=1547072087&width=1000&height=570&entry_id=1_gknidq66&videoid=ee6b8d51c7ebda0720a91025739',
```

Comment:
I actually ended up removing the test because I couldn't figure out how to make it work properly - should it be reading off a live webpage? Is there an example of something that does the right thing?

Reply:
There we go, figured it out.